nginx-controller/controller/controller.go

```diff
@@ -101,8 +101,19 @@ func NewLoadBalancerController(kubeClient kubernetes.Interface, resyncPeriod tim
 				glog.Infof("Ignoring Ingress %v based on Annotation %v", addIng.Name, ingressClassKey)
 				return
 			}
+
 			glog.V(3).Infof("Adding Ingress: %v", addIng.Name)
```

Comment:
105 and 106 should be places into the else block of `if addIng.Annotations["nginx.org/mergible-ingress-type"] == "minion"`. Otherwise, we will still be adding minions. Sames goes for `DeleteFunc` and `UpdateFunc`.